dashboard/app/assets/javascripts/dashboard.controller.js

```diff
@@ -0,0 +1,49 @@
+(function() {
+  'use strict';
+
+  angular.module('qat-dashboard').controller('DashboardController', [
+    'testExecutor',
+    function(testExecutor) {
+      var vm = this;
+
+      vm.testExecsMap = testExecutor.all;
+      vm.requestTestExec = requestTestExec;
+      vm.getDate = getDate;
+      vm.getStatusCssClass = getStatusCssClass;
```

Comment:
@mDibyo we probably want these at the end of the function. We should try to declare a function before it's used.